android/ObjectPermissionAdvanced/app/src/main/java/io/realm/chat/util/Constants.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.chat.util;
+
+final public class Constants {
+    private static final String INSTANCE_ADDRESS = "jarjar4.us1.cloud.realm.io";
```

Comment:
Perhaps add a comment to make be really clear this needs to be replaced by your own instance?